hoot-core/src/main/cpp/hoot/core/io/OsmGbdxJsonWriter.cpp

```diff
@@ -96,7 +100,24 @@ void OsmGbdxJsonWriter::_newOutputFile()
     close();
   }
 
-  QString url = _outputDir.filePath(UuidHelper::createUuid().toString().replace("{", "").replace("}", "") + ".json");
+  // The output has had a few changes.....
+//  QString url = _outputDir.filePath(UuidHelper::createUuid().toString().replace("{", "").replace("}", "") + ".json");
+//  QString url = _outputDir.filePath(QString("det_%1.json").arg(_fileNumber++,4,10,QChar('0')));
+//  QString url = _outputDir.filePath(QString("%1_%2.json").arg(_outputFileName).arg(_fileNumber++));
+  QString url = _outputDir.filePath(QString("%1_00_%2.json").arg(_outputFileName).arg(_fileNumber++));
+
+  // If the file exists, increment the middle _00_ in the name.
+  // NOTE: This assumes that there can be a maximum of 10 copies of a filename....
+  if (QFile::exists(url))
+  {
+//    LOG_ERROR("Clash: Orig Filename: " + url);
+    int inc = 0;
+    while (QFile::exists(url)) {
```

Comment:
c++ curmudgeon says this curly brace needs to be on its own line!

Reply:
Fixed